exporters/scss/src/js/constants/weight.ts

```diff
@@ -0,0 +1,31 @@
+// 1. Font Ebony has a different font weight mapping, so we remap these values directly
+
+export type DefaultWeightsType = typeof defaultWeights;
+const defaultWeights = {
+  thin: 100,
+  extralight: 200,
+  light: 300,
+  normal: 400,
+  regular: 400,
+  medium: 500,
+  semibold: 600,
+  bold: 700,
+  extrabold: 800,
+  black: 900,
+  extrablack: 950,
+};
+
+export type NormalizedWeightsType = {
+  default: DefaultWeightsType;
+  ebony: DefaultWeightsType;
+};
+
+export const normalizedWeights: NormalizedWeightsType = {
+  default: defaultWeights,
+  // 1.
```

Comment:
Why like this? I am expecting that the comment will be above the code I want to comment. Not to look for a reference somewhere else...